dotcom-rendering/src/components/StickyLiveblogAskWrapper.importable.tsx

```diff
@@ -1,31 +1,143 @@
 import { submitComponentEvent } from '../client/ophan/ophan';
 import { shouldHideSupportMessaging } from '../lib/contributions';
 import { useIsSignedIn } from '../lib/useAuthStatus';
 import { useCountryCode } from '../lib/useCountryCode';
 import { usePageViewId } from '../lib/usePageViewId';
+import type { TagType } from '../types/tag';
 import { useConfig } from './ConfigContext';
-import { GutterAsk } from './marketing/gutters/gutterAsk';
-import { props } from './marketing/gutters/utils/storybook';
-import type { ReactComponent } from './marketing/lib/ReactComponent';
-import {
-	addRegionIdAndTrackingParamsToSupportUrl,
-	createClickEventFromTracking,
-	createInsertEventFromTracking,
-} from './marketing/lib/tracking';
-
-// CSS Styling
-// -------------------------------------------
-
-// outer container handles the position and stickiness
+import { GutterAskWrapper } from './marketing/gutters/GutterAsk';
+
+interface GutterLiveblogAskBuilderProps {
+	shouldHideReaderRevenueOnArticle: boolean;
+	sectionId: string | undefined;
+	tags: TagType[];
+	contributionsServiceUrl: string;
+	countryCode: string;
+	pageViewId: string;
+	pageUrl: string;
+}
+
+const GutterLiveblogAskBuilder = ({
+	shouldHideReaderRevenueOnArticle,
+	sectionId,
+	tags,
+	contributionsServiceUrl,
+	countryCode,
+	pageViewId,
+	pageUrl,
+}: GutterLiveblogAskBuilderProps) => {
+	const [supportGutterResponse, setSupportGutterResponse] =
+		useState<ModuleData<GutterProps> | null>(null);
+
+	const { renderingTarget } = useConfig();
+	const isSignedIn = useIsSignedIn();
+
+	// get gutter props
+	useEffect((): void => {
+		if (isUndefined(countryCode) || isSignedIn === 'Pending') {
+			return;
+		}
+		const tagIds = tags.map((tag) => tag.id);
+
+		const hideSupportMessagingForUser =
+			shouldHideSupportMessaging(isSignedIn);
+		if (hideSupportMessagingForUser === 'Pending') {
+			// We don't yet know the user's supporter status
+			return;
+		}
+
+		const payload: GutterPayload = {
+			targeting: {
+				showSupportMessaging:
+					shouldHideSupportMessaging(isSignedIn) === false,
+				countryCode,
+				mvtId: Number(
+					getCookie({ name: 'GU_mvt_id', shouldMemoize: true }),
+				),
+				isSignedIn,
+				tagIds,
+				sectionId,
+			},
+		};
+
+		getGutterLiveblog(contributionsServiceUrl, payload)
+			.then((response: ModuleDataResponse<GutterProps>) => {
+				if (!response.data) {
+					throw new Error(
+						'No response.data returned from getGutterLiveblog',
+					); // TODO: appropriate?
+				}
+
+				const { module } = response.data;
+				setSupportGutterResponse(module);
+			})
+			.catch((error) => {
+				const msg = `Error: ${String(error)}`;
+				console.log(msg);
+				// TODO: where to log this?
+			});
+	}, [
+		countryCode,
+		isSignedIn,
+		contributionsServiceUrl,
+		pageViewId,
+		pageUrl,
+		sectionId,
+		tags,
+	]);
+
+	const canShow = !shouldHideReaderRevenueOnArticle; // TODO: anything else?
+
+	if (supportGutterResponse) {
```

Comment:
we could check `canShow` here

Reply:
Returning if shouldHideReaderRevenue before calling the builder now so not calling the API